### PR TITLE
Issue#13

### PR DESCRIPTION
**Pull request by pchawlaji**
## Description
Added Documentation.js file with basic skeleton.

## Related Issues
https://github.com/Curtis-Thomas/free-api-list-with-react/issues/13

## Checklist

- [X] I have tested my changes locally.
- [X] I have added necessary documentation or updated existing documentation.
- [X] My code follows the project's coding conventions and style guidelines.
- [X] All tests pass successfully.
- [X] I have added/updated relevant unit tests.

## Additional Notes

Include any additional notes or context about your pull request.

### CODE REVIEW

File: src/documentation/Documentation.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
```
Comment (Curtis-Thomas) [inline src/documentation/Documentation.js:2]: Insert:

import { Box, Typography } from "@mui/material";

File: src/documentation/Documentation.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * The Documentation component returns a simple div element containing an h1 heading. 
+ *  Add your documentation content or components within the div as needed.
+*/
+const Documentation = () => {
+    return (
+        <div>
```
Comment (Curtis-Thomas) [inline src/documentation/Documentation.js:10]: Swap to:

`<Box>`

File: src/documentation/Documentation.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            <h1>Documentation Page</h1>
```
Comment (Curtis-Thomas) [inline src/documentation/Documentation.js:11]: Swap opening tag to:

`<Typography variant="h2">`

and closing tag to:

`</Typography>`

File: src/documentation/Documentation.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            {/* Add your content here */}
+        </div>
```
Comment (Curtis-Thomas) [inline src/documentation/Documentation.js:13]: Swap to:

`</Box>`

**Review by Curtis-Thomas — CHANGES_REQUESTED** [review]
Hi!

Thank you for the great work, just a few small changes because the project is using Material UI which uses slightly different tags :)

**Comment by pchawlaji** [comment]
Hi @Curtis-Thomas ,
Thank you for taking time to review my request. I really appreciate the feedback given. I have made the requisite changes.

**Review by Curtis-Thomas — APPROVED** [review]
Thank you :)

**Comment by pchawlaji** [comment]
@Curtis-Thomas Thanks a ton... That was my first merge. :)

**Pull request merged**